src/client.ts

```diff
@@ -493,15 +497,19 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
       throw createAndEmitSdkError.call(this, SdkErrorTypes.http, err.message, err);
     });
     this.station = body;
-    this.logger.debug('Fetched user station', body, true); // don't log PIIdefineProperty
+    this.logger.debug('Fetched user station', body, true); // don't log PII
```

Comment:
I think here I'd rather log an `info` with the stationId and lineAppearance and whether or not it's a webrtc phone.